module/witcher.js

```diff
@@ -1,5 +1,5 @@
 export function getRandomInt(max) {
-    return Math.floor(Math.random() * (max + 1)) + 1;
+		return Math.floor(Math.random() * (max + 1)) + 1;
```

Comment:
Feels like a lot of unnecessary "tabs" :P

Reply:
Yeah you might need to change the configs of your IDE, it's seems to put a lot of uncessary tabs

Reply:
Oops, I've replaced two whitespaces instead of four. Should be good now.